docs/windows.md

```diff
@@ -0,0 +1,195 @@
+
+Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided by kubernetes. The third component
+[`wins`](https://github.com/rancher/wins) is used to run kube-proxy and Antrea-Agent on Windows host inside the Windows
+container.
+```
+Example:
+curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/PrepareNode.ps1
+.\PrepareNode.ps1 -KubernetesVersion v1.17.0
+```
+Then, set the Node IP used by kubelet.
+Open file `/var/lib/kubelet/kubeadm-flags.env`:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1"
+```
+Append "--node-ip=$NODE_IP" arg at the end of params. Replace `$NODE_IP` with the address for kubelet. It should look
+like:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1 --node-ip=$NODE_IP"```
+```
+
+3.Prepare network adapter for kubernetes service.
+Kube-proxy need a network adapter to configure kubernetes service IPs and use the adapter for network proxy.
+```
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Prepare-ServiceInterface.ps1
+.\Prepare-ServiceInterface.ps1
+```
+
+3.Run kubeadm to join the node.
+
+On windows worker node, use the command that was given to you when you ran `kubeadm init` on a control plane host to
+join the node to cluster.
+
+If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command`
+(on a control plane host) to generate a new token and join command.
+```
+Example:
+kubeadm join 192.168.101.5:6443 --token tdp0jt.rshv3uobkuoobb4v     --discovery-token-ca-cert-hash sha256:84a163e57bf470f18565e44eaa2a657bed4da9748b441e9643ac856a274a30b9
+```
+
+#### Verify your installation
+There will be temporary network interruption on windows worker node on the first startup of Antrea Agent. It's because
+Antrea Agent will set the OVS to take over the host network. After that you should be able to view the Windows Nodes and
+Pods in your cluster by running:
+
+```
+# Show nodes
+kubectl get nodes -o wide -nkube-system
+NAME                           STATUS   ROLES    AGE    VERSION   INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                                  KERNEL-VERSION     CONTAINER-RUNTIME
+win-5akrf2tpq91                Ready    <none>   1h   v1.17.0   10.176.27.150   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.1158    docker://19.3.5
+win-5akrf2tpq92                Ready    <none>   1h   v1.17.0   10.176.27.197   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.1158     docker://19.3.5
+
+# Show Antrea Agent and kube-proxy pods
+kubectl get pods -o wide -nkube-system | grep windows
+antrea-agent-windows-6hvkw                             1/1     Running     0          100s
+kube-proxy-windows-2d45w                               1/1     Running     0          102s
+```
+
+
+## Common issues
```

Comment:
Known issues?

Reply:
We should add NetworkPolicy for Service traffic here if it cannot be fixed in 0.7.0.

Reply:
Yes, we need to add Service issue here if we cannot fix it before release.

Done.

Reply:
Yes, done.